src/index-jQuery.js

```diff
@@ -0,0 +1,48 @@
+// const BowlingScorecard = require('./BowlingScorecard.js');
+// import BowlingScorecard from
+
+let scorecard = new BowlingScorecard()
+// scorecard.enterRoll(5)
+// scorecard.enterRoll(4)
+// scorecard.enterRoll(5)
+// scorecard.enterRoll(5)
+// scorecard.enterRoll(5)
+// scorecard.enterRoll(5)
+
+$( document ).ready(function() {
+
+  // alert( "Welcome to the scorecard!");
+  // console.log(scorecard._tidyDisplay(5))
+  displayScorecard()
+
+  $( '#enterRoll-form' ).submit(function( event ) {
+    let score = $("#rollScore").val()
+    console.log(scorecard.enterRoll(score))
+    displayScorecard()
+    event.preventDefault();
+  });
```

Comment:
What does preventDefault do here?

Reply:
Prevent Default stops the normal function of clicking submit on the form (which seems to refresh the page if not given a direction) so you need preventDefault to stop the standard function of the submit button and allow the javascript methods to resolve and stay on the page.

Reply:
I would have thought you didn't need it as your form doesn't have an action or a method. It would be worth trying without the prevent default in, to see if it still works as intended.